documentation/troubleshooting/ipi-install-upstream-troubleshooting.adoc

```diff
@@ -16,10 +16,21 @@ Deployment Integration Team
 :sectlinks:
 :icons: font
 :product-title: OpenShift Container Platform
+:op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
+:op-system: RHCOS
+:asb-name: OpenShift Ansible Broker
+:tsb-name: Template Service Broker
+:kebab: image:kebab.png[title="Options menu"]
+:rh-openstack-first: Red Hat OpenStack Platform (RHOSP)
+:rh-openstack: RHOSP
+:cloud-redhat-com: Red Hat OpenShift Cluster Manager
```

Comment:
Seems like we are using `op-system-first` and `op-system` but none of the others. I'm thinking not working add the extras.

Reply:
Same comment as above, left to avoid 'future problems' but can be removed

Reply:
What I think they do is they use a standard list of variables for all their docs even if not used. In our case at least for this troubleshootig doc, I don't see us using any of them and rather just include on a needed basis as it just looks odd to me to include a variable for something we don't use. Of course if that changes, we can add in the variable as needed.